src/plone/distribution/exportimport/exportimport.py

```diff
@@ -6,16 +6,36 @@
 from pathlib import Path
 from plone import api
 from plone.distribution.api import distribution as dist_api
+from plone.distribution.core import Distribution
 from plone.distribution.exportimport.interfaces import IDistributionBlobsMarker
 from Products.Five import BrowserView
 from typing import List
 from zope.interface import alsoProvides
 
+import os
+
 
 logger = getLogger(__name__)
 
 
+def _filter_devel_distributions(name: str = "") -> List[Distribution]:
+    """List of distributions still in development."""
+    distributions = []
+    devel_distributions = os.environ.get("DEVELOP_DISTRIBUTIONS", "")
+    for dist_name in devel_distributions.split(","):
+        if name and dist_name != name:
+            continue
+        distribution = dist_api.get(name=dist_name)
+        if distribution:
+            distributions.append(distribution)
+    return distributions
```

Comment:
I think I'm missing the reason for filtering the distributions that can be exported like this. What if a distribution is not "still in development" but needs a trivial fix for a typo or something...then wouldn't it be desired to export it?

Reply:
Right now, we will list all available distributions, including the default ones (Volto and Classic), whether installed in editable mode or not. 
Example: When developing `collective.ploneintranet`, using the released version of `plone.distribution`, we should see only `collective.ploneintranet` distributions, not the default one's during the export process.